smg-automotive/components-pkg
# fix: update vehicle icons

References [jira ticket](https://autoricardo.atlassian.net/browse/DM-876)

## Motivation and context
Vehicle icons are updated by designers. The goal of the PR is to have up-to-date vehicle icons.

## Before

Old vehicle icons:
<img width="1717" alt="Screenshot 2022-12-12 at 12 27 44" src="https://user-images.githubusercontent.com/55357316/207034228-bb119ddf-015a-4925-8583-a6bfd79bb560.png">


## After
New vehicle icons according to the design:
<img width="1709" alt="Screenshot 2022-12-12 at 12 37 29" src="https://user-images.githubusercontent.com/55357316/207035956-7753c126-12f9-45d7-9994-d78f1f1fb000.png">


## How to test
Go to: https://fix-vehicle-icons.components-pkg.branch.autoscout24.ch/?path=/story/foundations-icons--icons
Check icons
